src/arviz_stats/summary.py

```diff
@@ -68,11 +68,11 @@ def summary(
 
     See Also
     --------
-    arviz.rhat : Compute estimate of rank normalized split R-hat for a set of traces.
-    arviz.ess : Calculate the effective sample size of a set of traces.
-    arviz.mcse : Calculate Markov Chain Standard Error statistic.
-    plot_ess : Plot quantile, local or evolution of effective sample sizes (ESS).
-    plot_mcse : Plot quantile, local or evolution of Markov Chain Standard Error (MCSE).
+    arviz_stats.rhat : Compute estimate of rank normalized split R-hat for a set of traces.
+    arviz_stats.ess : Calculate the effective sample size of a set of traces.
+    arviz_stats.mcse : Calculate Markov Chain Standard Error statistic.
```

Comment:
We could probably drop the arviz_stats here. I don't see anywhere else where we do that. This should still link to the relevant functions in the API. 

```suggestion
    rhat : Compute estimate of rank normalized split R-hat for a set of traces.
    ess : Calculate the effective sample size of a set of traces.
    mcse : Calculate Markov Chain Standard Error statistic.
```